scripts/check_licenses.py

```diff
@@ -0,0 +1,92 @@
+"""Check if license fields are valid in all records."""
+
+import asyncio
+import json
+import logging
+import os
+import pathlib
+import time
+
+VALID_LICENSE_IDENTIFIERS = [
```

Comment:
<!-- Praises highlight something positive. Try to leave at least one of these comments per review. Do not leave false praise (which can actually be damaging). Do look for something to sincerely praise.  -->
**praise:** ‏I appreciate that you included the checker tool together with fixture changes :+1: